Gemfile

```diff
@@ -3,15 +3,15 @@
 source 'https://rubygems.org'
 ruby IO.read(File.expand_path('.ruby-version', __dir__)).strip
 
-gem 'bootstrap', '~> 4.0'
-gem 'friendly_id', '~> 5.1'
+gem 'bootstrap'
+gem 'friendly_id'
 gem 'haml'
 gem 'haml-rails'
 gem 'icalendar'
 gem 'mysql2'
 gem 'prawn'
 gem 'prawn-table'
-gem 'rails', '~> 6.1.3'
+gem 'rails', '~> 6.1'
```

Comment:
Rails is never going to release 6.2, but if they did, it would be a manual process to upgrade. Rails uses a "modified" version of semver that's essentially REALLY_MAJOR.MAJOR.MINOR.PATCH